deployments/vanilla/terraform/Makefile

```diff
@@ -0,0 +1,42 @@
+SHELL := /bin/bash # Use bash syntax
+
+deploy: init deploy_vpc deploy_eks_blueprints deploy_eks_blueprints_k8s_addons deploy_kubeflow_components
```

Comment:
- can we do multi line instead of single line?
- nit: use - instead of _ in all commands
- I know destroy makes more sense for terraform but can we use deploy and destroy to be in sync with other options?

Reply:
done